packages/cfd/src/sass/cfd-dashboard.scss

```diff
@@ -1303,15 +1303,10 @@
             align-items: center;
             gap: 0.8rem;
         }
-        &-text {
-            overflow: hidden;
-        }
         &-action {
-            padding: 0;
-            height: 1.5rem;
-            .dc-btn__icon {
-                padding-right: 0;
-            }
+            text-decoration: underline;
```

Comment:
Question: Why do these styles from cfd package still apply ? 🤔

Reply:
The styles need to be added in the trader's hub but I keep the CFD package styles updated until we completely remove this package. Just keeping them in sync. The CFD package styles do not apply though.

Reply:
@hirad-deriv if the styles are not being used then just remove them.